### DISCUSSION TITLE
ScriptReference/QualitySettings

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#224] [Notes]
# ScriptReference/QualitySettings

https://docs.unity3d.com/ScriptReference/QualitySettings.html

**Comment by mvi** [comment]
To access the other quality settings as well as what the per platform defaults are you can't use the static properties and methods, instead you must load the QualitySettings Unity object via the AssetDatabase then work with it via SerializedObject or Reflection. See [QualitySettingsEditor](https://github.com/Unity-Technologies/UnityCsReference/blob/master/Editor/Mono/Inspector/QualitySettingsEditor.cs) for reference on how the Unity Editor deals with these properties.  

```csharp
QualitySettings qualitySettings = AssetDatabase.LoadAssetAtPath<QualitySettings>("ProjectSettings/QualitySettings.asset");  
SerializedObject serializedObject = new SerializedObject(qualitySettings);
```
